### PR TITLE
Merge recent changes from dev to master

### PR DESCRIPTION
**Pull request by nibinqtl**
Let's do this bi-weekly.

### CODE REVIEW

File: docs/recipes/aws-bucket.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+      }
+ ```
+
+From now, you can edit this script and setup any logic you want. Having all about there one small thing remaining for this property - Add the Certificate. Choose a certificate we have created and deployed to Staging previously.
+
+<p align=center><img src="/docs/resources/images/recipes/aws-bucket/create-property-tls.png" alt="create bucket" width="600"></p>
+	
+Click “Validate & Save” to check if everything is configured correctly.
+
+Deploy a Property to Staging.
```
Comment (burtonquantil) [inline docs/recipes/aws-bucket.md:118]: Is this supposed to formatted differently as the title of a section?

Suggestion:  Deploy the property to staging.

File: docs/recipes/aws-bucket.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+
+Suppose no issues occurred during the “Validate & Save” - congratulations. Now we can deploy property and test our Staging deployment.
```
Comment (burtonquantil) [inline docs/recipes/aws-bucket.md:120]: Suggestion:  If the property was saved and validated without any issue, we can now deploy the property and test our staging deployment.

File: docs/recipes/aws-bucket.md [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+
+<p align=center><img src="/docs/resources/images/recipes/aws-bucket/deploy-to-staging.png" alt="create bucket" width="500"></p>
+
+#### Testing(Staging) 
+
+To test staging deployments, we should know the staging servers IP addresses:
```
Comment (burtonquantil) [inline docs/recipes/aws-bucket.md:126]: "we should get the staging servers' IP addresses"

File: docs/recipes/aws-bucket.md [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+```bash
+$ dig staging.qtlgslb.com
+...
+;; ANSWER SECTION:
+staging.qtlgslb.com.    59  IN  A   163.171.228.89
+staging.qtlgslb.com.    59  IN  A   163.171.230.47
+...
+```
+
+Now, let's pick one of those addresses returned above and check if everything we have setup and deployed previously works.
```
Comment (burtonquantil) [inline docs/recipes/aws-bucket.md:137]: Suggestion: Replace with "Now, let's pick one of those addresses and check if everything we have previously set up and deployed  works."

File: docs/recipes/aws-bucket.md [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+etag: "fcaebe20d9b3e73086dcb0ba5def8e52"
+x-cache-status: MISS
+x-qtl-request-id: 4c394e8fd8cc514c5f1e437ac3daf969
+x-via: 2.0 na-us-iad1-cache-0003 [MISS 200]
+age: 19
+server: QTL_Cache/1.2.03
+accept-ranges: bytes
+```
+
+By requesting the same item again, we will see that now cache status is **HIT**
```
Comment (burtonquantil) [inline docs/recipes/aws-bucket.md:157]: Suggestion:  "By requesting the same item again, we will see the cache status change to **HIT**

File: docs/recipes/aws-bucket.md [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+etag: "fcaebe20d9b3e73086dcb0ba5def8e52"
+x-cache-status: HIT
+x-qtl-request-id: dfa55f3549ee4fd0824472f83d06c6c5
+x-via: 2.0 na-us-iad1-cache-0003 [HIT]
+age: 13
+server: QTL_Cache/1.2.03
+accept-ranges: bytes
+```
+
+Everything works as expected, so let's deploy everything to production.
```
Comment (burtonquantil) [inline docs/recipes/aws-bucket.md:177]: remove the second "everything"

File: docs/recipes/aws-bucket.md [hunk 7]
```diff
@@ -0,0 +1,264 @@
...
+
+#### Deployment to Production
+
+To deploy the configuration to production, we should do it both in **"Certificates"** and **"Edge Configurations"**. Navigate to **"Certificates"** first, select certificate, set *Deployment destination* to *Production* and click **"Deploy Configuration"**.
```
Comment (burtonquantil) [inline docs/recipes/aws-bucket.md:181]: I suggest changing the first sentence to:  "To deploy the configuration to production, we must deploy both the certificate and the property.  Navigate to **"Certificates"** first. Select the certificate.   In the *Deployment Destination" field, choose "Production", and click **"Deploy Configuration"**.

@marcfishelquantil

File: docs/recipes/aws-bucket.md [hunk 8]
```diff
@@ -0,0 +1,264 @@
...
+
+<p align=center><img src="/docs/resources/images/recipes/aws-bucket/deploy-to-production.png" alt="create bucket" width="500"></p>
+
+Then navigate to **"Edge Configurations"** and do the same with your property. 
+
+#### Testing(Production)
+
+Before we can try it in production, we should Create/Change the CNAME record of your DNS and point it to the *Edge Hostname* we have created in the beginning(`files-waytoo-digital.qtlcdn.com`)
```
Comment (burtonquantil) [inline docs/recipes/aws-bucket.md:189]: Suggestion: Before we can try it in production, we must modify our DNS by creating or changing the CNAME record for files.waytoo.digital to point it to the edge hostname we created earlier, `files-waytoo-digital.qtlcdn.com`.

File: docs/recipes/aws-bucket.md [hunk 9]
```diff
@@ -0,0 +1,264 @@
...
+```bash
+❯ dig files.waytoo.digital
+...
+;; ANSWER SECTION:
+files.waytoo.digital.	299	IN	CNAME	files-waytoo-digital.qtlcdn.com.
+files-waytoo-digital.qtlcdn.com. 19 IN	A	14.0.119.186
+...
+```
+
+Actually, you also can do use the production IP address to check if everything is ok before you will change the DNS record. Just like that:
```
Comment (burtonquantil) [inline docs/recipes/aws-bucket.md:200]: Suggestion:  You can also use the cURL utility to check that everything is OK before modifying your DNS server:

File: docs/recipes/aws-bucket.md [hunk 10]
```diff
@@ -0,0 +1,264 @@
...
+etag: "fcaebe20d9b3e73086dcb0ba5def8e52"
+x-cache-status: HIT
+x-qtl-request-id: 16a3be4dfdc381e44b90c86bec087f47
+x-via: 2.0 as-kr-icn1-cache-0003 [HIT]
+age: 5575
+server: QTL_Cache/1.2.03
+accept-ranges: bytes
+```
+
+Congratulations. You've just accelerated Amazon S3 Bucket with **CDN360**
```
Comment (burtonquantil) [inline docs/recipes/aws-bucket.md:220]: Congratulations! You've just accelerated an Amazon S3 bucket with **CDN360**.

File: docs/recipes/aws-bucket.md [hunk 11]
```diff
@@ -0,0 +1,264 @@
...
+
+### Accessing Private Buckets
+
+Now, let's block the public access to our bucket and setup a CDN360 to access it.
```
Comment (burtonquantil) [inline docs/recipes/aws-bucket.md:224]: Now, let's block public access to our bucket and set up CDN360 to access it.

File: docs/recipes/aws-bucket.md [hunk 12]
```diff
@@ -0,0 +1,264 @@
...
+
+#### Block public access
+
+Previously we have made our bucket public. Now, in bucket settings, we should set the default permissions back to make the bucket private again.
```
Comment (burtonquantil) [inline docs/recipes/aws-bucket.md:228]: Suggestion:  "Previously, we made our bucket public.  Now in AWS' bucket settings, we should set the default permission back to make the bucket private again."

File: docs/recipes/aws-bucket.md [hunk 13]
```diff
@@ -0,0 +1,264 @@
...
+
+<p align=center><img src="/docs/resources/images/recipes/aws-bucket/bucket-permissions-private.png" alt="create bucket" width="720"></p>
+
+#### Enable Origin Authentication
+
+Now we can edit our property settings and setup the `Origin Authentication for S3`. Before doing that, make sure you know: *1) AWS Bucket region*, *2) AWS Acces Key*, *3) AWS Access Secret*.
```
Comment (burtonquantil) [inline docs/recipes/aws-bucket.md:234]: Change "setup" to "set up".
Change "Acces" to "Access".

(Is it obvious to the user how they get those three pieces of information?)

File: docs/recipes/aws-bucket.md [hunk 14]
```diff
@@ -0,0 +1,264 @@
...
+
+All right. Now go to **"Edge Configurations** then choose the property we configured before.
```
Comment (burtonquantil) [inline docs/recipes/aws-bucket.md:236]: Change to:  Alright, now go to **Edge Configurations** and choose the property we configured before.

File: docs/recipes/aws-bucket.md [hunk 15]
```diff
@@ -0,0 +1,264 @@
...
+
+<p align=center><img src="/docs/resources/images/recipes/aws-bucket/modify-property-clone.png" alt="create bucket" width="720"></p>
+
+Click *"Clone"* to create a new configuration version. Now we can edit it. Click on *"Edit"* and then in *"Advanced Settings"* set the Authentication Method to *"AWS S3"* and fill the required fields as follows.
```
Comment (burtonquantil) [inline docs/recipes/aws-bucket.md:240]: Change "fill the required" to "fill in the required"

**Review by burtonquantil — CHANGES_REQUESTED** [review]
Marc and I both need to fix links to the origin_fast_route directive.
aws-bucket.md needs a lot of work.

**Comment by snikitin-qtl** [comment]
There the PR to address AWS page's language issues -> https://github.com/mileweb/docs-content-cdn/pull/109

**Review by burtonquantil — APPROVED** [review]
OK to merge

**Pull request merged**